src/components/DeleteModalUI.tsx

```diff
@@ -0,0 +1,46 @@
+const MODAL_MESSAGES = {
+  confirmDeleteAll: 'Are you sure you want to delete all tasks?',
+  noTasksToDelete: 'No tasks to delete.',
+  closeButton: 'Close',
+  deleteButton: 'Delete',
+};
+
+interface DeleteModalUIProps {
+  handleDeleteAll: () => Promise<void>;
+  handleCloseModal: () => void;
+  tasksExist: boolean;
```

Comment:
Nitpick 🪡:  prefer names that start with `has` or `is` for booleans. `hasTasks` for example would be better here.

Reply:
updated to `hasTasks`